hack/vendor/olm-test-script/e2e-olm.sh

```diff
@@ -70,6 +71,26 @@ if [ -n "${OPERATOR_IMAGE:-}" ] ; then
   CSV=$(echo "$CSV" | sed -e "s~image:.*~image: ${OPERATOR_IMAGE}\n~" | indent ApiVersion)
 fi
 
+ci_image_prefix="registry.svc.ci.openshift.org/origin/${VERSION}"
+
+# Use latest origin-aggregated-logging
+if [ -n "${IMAGE_FORMAT:-}" ] ; then
```

Comment:
We need to figure out a way to pass this in. Ideally this script can be contributed back to be generic for operators and this breaks that: https://github.com/shawn-hurley/olm-test-script

Reply:
Isn't this already passed though? it came from /hack/utils

Reply:
`${IMAGE_FORMAT:-}` comes from the CI environment - so if you are trying to test outside of the CI environment, you'll have to use the alternate methods as in https://github.com/openshift/origin-aggregated-logging/blob/master/hack/deploy-logging.sh#L187-L192